.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,104 @@
+name: Mac and Windows Builds
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+        python-version: ['3.5', '3.6', '3.7']  # We build 3.5 on Azure Pipelines
```

Comment:
You mean we build 3.8 on Azure pipelines?

Reply:
My bad, that was a comment from when I couldn't get 3.5 to work properly on GitHub. I will remove that

Reply:
Also, 3.8 PyPI distributions will likely git built on GitHub (at least for mac and windows). I'd like to keep it as:

- CircleCI - Linux
- Github Actions - Windows and Max
- Azure Pipelines - Conda